flybywiresim/aircraft
# feat(eis): move trans alt/lvl to arinc bus

Fixes #[issue_no]

## Summary of Changes

Utilize the FMGC DMC bus to transmit transition altitude and transition level. This should not have any visible effect. However, it gets rid of the undocumented `L:AIRLINER_TRANS_ALT` and `L:AIRLINER_APPR_TRANS_ALT` variables.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Check that the QNH/STD indication on the PFD still blinks once the aircraft crosses the transition altitude/level in the climb/descent.
- Check that there is no blinking when the corresponding fields are empty in the MCDU.



## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
